### ISSUE TITLE
Add term triangular shaped nail

### ISSUE THREAD

**Issue by lisannesw** [obophenotype/human-phenotype-ontology#10407]
**Preferred term label:**
Triangular shaped nail

**Synonyms**

**Definition (free text, please give PubMed ID)**
Nail so wide as long with a distal part presenting an acute angle. PMID: 27660117
![image](https://github.com/obophenotype/human-phenotype-ontology/assets/160679899/7abbff53-1e06-414a-bf68-1a236c8de4cb)

**Parent term (use hpo.jax.org/app)**
Aplasia/Hypoplasia of the nails HP:0008386

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
for example: PMID: 27660117, PMID: 20579456, PMID25251565_2

**Your nano-attribution (ORCID)**
Andreas Zankl: 0000-0001-8612-1062, @azankl
Lisanne Wit: 0009-0007-1020-5412

**Comment by pnrobinson** [comment]
Adding new term: 
- ID: HP:6000649
- label: Triangular nail
- parents: [Abnormality of the nail (HP:0001597)](https://hpo.jax.org/app/browse/term/HP:0001597)
- pmid: PMID:27660117|PMID:20579456|PMID:25251565
- def: Nail so wide as long with a distal part presenting an acute angle.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10407
- synonyms: Triangular shaped nail(NONE)
